web/src/Search.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import './App.css';
```

Comment:
These are no longer styles for `App`, but for `Search` - can you change filename?